packages/acs-ui-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internal/acs-ui-common",
-  "version": "1.0.1-beta.2",
+  "version": "1.1.0",
```

Comment:
@prprabhu-ms remember when you mentioned before the `prerelease` option suddenly wasn't available when you did rush changelog -- did we decide this was because the package didn't have a prerelease suffix? (wondering if this will mean the prerelease option will vanish in rush changelog)

Reply:
On completing this PR, there will be a time (until we bump to beta again), that inner-loop devs cannot choose prerelease as the changelog option -- blocking PRs. Not sure what to do about this, unless we immedaitely make this branch have 1.1.-beta.0

Reply:
This means the prerelesse option will be disabled when beachball detects that cutrent branch is not beta? I thought that option was always there, good to learn

Reply:
This version will be bumped again in next beta release as what we previous designed - 1.1.1-beta

But as what James mentioned, we need to make sure prelease option won't be taken away when we run 'rush changelog'

Reply:
I support bumping to `1.1.0-beta.0` immediately.

Reply:
`1.1.1-beta.0` *

Reply:
Todo
- Replace version 1.1.0 with 1.1.1-beta.0
- Merge with main